src/components/menu-bar/menu-bar.css

```diff
@@ -57,11 +57,6 @@
     width: $language-selector-width;
 }
 
-.menu {
-    z-index: $z-index-menu-bar;
-    top: $menu-bar-height;
-}
-
```

Comment:
This belongs here, for the same reason — the menu bar is the one positioning its menus, so it should control these styles.  Even though you've refactored them into their own files, they should receive classnames from the menubar component (or other components using them) which apply the positioning styles.

Reply:
Done

Reply:
Here I was suggesting to bring back the positioning styles that have been removed.

Reply:
I brought these lines back, just moved them to elsewhere in this file.

`menu-bar.css` line 103:
```
.menu-bar-menu {
    margin-top: $menu-bar-height;
    z-index: $z-index-menu-bar;
}
```